precovery/frame_db.py

```diff
@@ -547,7 +546,12 @@ def load_hdf5(
             chunksize=chunksize,
         ):
             observations = [Observation.from_srcobs(o) for o in src_frame.observations]
-            data_uri, offset, length = self.store_observations(observations)
+            year_month_str = "-".join(
+                Time(src_frame.mjd, format="mjd", scale="utc").isot.split("-")[:2]
```

Comment:
I think we could also do 
`Time(src_frame.mjd, format="mjd", scale="utc").isot.format_string("%YYYY-%MM")`

Not really important I just hate grabbing slices due to edge cases.